package oops_Basic;

public class Exception_Handling 
{
	public static void main(String args[]) throws InterruptedException 
	{
		try 
		{
			//final int i=0,j=1,k;
			
			//k=i/j;
			//System.out.println("******************* "+ k);
			
			/*
			 * int a[]= {1,2,3, 4,5,6};
			 * 
			 * 
			 * System.out.println(a[9]);
			 */
//			
//			int i;
//			System.out.println("**************" +i);
//			
		}
		catch(Exception e) 
		{
			throw e;
			
		}
		finally 
		{
			System.out.println("this is finally block");
		}
		
		
		
	}

}
